conf/datapaths/cannon_datapaths.yaml

```diff
@@ -1,10 +1,7 @@
 # if the files are stored in cannon, then the paths are: 
 input:
-  uds_raw_xwalks: null
-  #/n/holyscratch01/dominici_lab/Lab/uds_raw_xwalks
-  #zcta_shapefiles: /n/dominici_lab/lab/data/exposures/exposure/zcta_shapefiles_tigerline
+  uds_raw_xwalks: /n/dominici_lab/lab/lego/geoboundaries/zip2zcta__uds/raw
 intermediate:
-  uds_clean_xwalk: null
-  #/n/holyscratch01/dominici_lab/Lab/uds_clean_xwalk
+  uds_clean_xwalk: /n/dominici_lab/lab/lego/geoboundaries/zip2zcta__uds/.intermediate
```

Comment:
I am not familiar with cannon but just a heads up that here there is a dot before intermediate, it's surely intentional but I just prefer to flag it

Reply:
yep! this is intentionally a hidden folder. Thanks for noting!